inventory/sample/group_vars/k8s-cluster/addons.yml

```diff
@@ -115,6 +115,7 @@ ingress_publish_status_address: ""
 ingress_ambassador_enabled: false
 # ingress_ambassador_namespace: "ambassador"
 # ingress_ambassador_version: "*"
+# ingress_ambassador_multi_namespace: false
```

Comment:
How about `ingress_ambassador_multi_namespaces` with `s` ?

Reply:
I'm fine with that.
I'll update the PR in a bit.